aptos-move/vm-genesis/src/lib.rs

```diff
@@ -526,11 +558,15 @@ pub fn generate_test_genesis(
         OnChainConsensusConfig::default(),
         ChainId::test(),
         &GenesisConfigurations {
-            epoch_duration_secs: 86400,
+            allow_new_validators: false,
+            epoch_duration_secs: 3600,
             min_stake: 0,
+            min_voting_threshold: 0,
             max_stake: 1000000,
-            recurring_lockup_duration_secs: 1,
-            allow_new_validators: false,
+            recurring_lockup_duration_secs: 7200,
+            required_proposer_stake: 0,
+            rewards_apy_percentage: 10,
+            voting_duration_secs: 3600,
```

Comment:
So, we've got a bunch of defaults copy and pasted everywhere.  Can we make constants and reuse them across the crates

Reply:
There are only 2 places: here and in the genesis builder. The other places are in the yaml config for Forge or testnet genesis, which can't reuse any constants anyway I think. For now I'm leaning toward just leaving these 2 values be different as the genesis builder might be used differently from the values used for tests defined here.